lib/demangler/win32.ts

```diff
@@ -144,10 +153,10 @@ export class Win32Demangler extends CppDemangler {
         }
 
         await Promise.all(commandLineArray.map(demangleSingleSet));
-        return translations;
+        return translations as unknown as UnprocessedExecResult;
```

Comment:
All other demanglers return a `Promise<UnprocessedExecResult>` except the windows demanglers, which return a `Record<string, string>`, but I'm not sure why they work while doing this. `processOutput` still takes a `UnprocessedExecResult`.

Reply:
looks like they have a different implementations for both functions, might want to just make different functions instead 
of override

Reply:
(https://github.com/compiler-explorer/compiler-explorer/pull/4607/files/3f1bd9175e38838819686ddda43fde0f7dd5fad8#diff-c044a135f1c5bbd3f3c4779dbb3b970f0e532e7fe71491b261daf5367e926064R86 definitely used as an array, but obviously objects can also be indexed by name, so the ts compiler can't give an error on it)

Reply:
I think you're right, just making a different function is probably the only solution here